src/app/return-credentials/return-credentials.component.html

```diff
@@ -8,7 +8,8 @@
         routerLinkActive="mat-tab-link-active" #rla="routerLinkActive"
         [active]="rla.isActive"
         (click)="activeLinkIndex = i"
-        [routerLinkActiveOptions]="{exact: true}">
+        [routerLinkActiveOptions]="{exact: true}"
+        disabled="{{checkClientType(link.label)}}" >
```

Comment:
better to use [] syntax rather than {{}} when invoking function on property binding

Reply:
it is not a property I think